src/js/components/explorer/detail/header/DetailHeader.jsx

```diff
@@ -63,31 +63,47 @@ const dataType = (type, parent) => {
 
 const heading = (type, title, id) => {
     if (type === 'Federal Account') {
+        let header = (
+            <a
+                href={`/#/federal_account/${id}`}
+                className="detail-header__title-link"
+                onClick={exitExplorer.bind(null, `/federal_account/${id}`)}>
+                {title}
+            </a>);
+        if (title === "Unreported Data") {
```

Comment:
We shouldn't need to check for the unreported screen here because Federal Account is not a root-level type (Budget Function, Agency, or Object Class)